src/app/not-found.tsx

```diff
@@ -1,5 +1,45 @@
+import Image from "next/image";
+import Button from "@/components/Button";
+import Logo from "@/public/Footer.webp";
+import Link from "next/link";
+import { links } from "@/data/Footer";
+
 const NotFound = () => {
-  return <div>404 Page Not Found</div>;
+  return (
+    <div className="flex h-[820px] w-full flex-col items-center justify-center bg-gradient-to-b from-hlg-dark-gray via-hlg-dark-gray to-hlg-red-200">
+      <Image
+        src={Logo}
+        alt="Highlander Gloves Logo"
+        className="absolute left-8 size-[620px] translate-y-20 transform items-end"
+      />
+      <div className="flex flex-col items-center justify-center pt-20">
+        <div className="relative flex items-center justify-center rounded-3xl bg-hlg-red-200 px-28 py-8 text-3xl">
+          <p className="absolute translate-x-2 translate-y-2 self-center font-archivo-black text-9xl text-black">
+            404
+          </p>
+          <p className="relative font-archivo-black text-9xl">404</p>
+        </div>
+        <p className="relative pb-10 pt-4 font-archivo-narrow text-[130px]">
```

Comment:
use `rem` for `text` size

Reply:
do you mean like this? `text-[8.125rem]`

Reply:
yes or use `em` up to you